test/models/Company/Repository.test.ts

```diff
@@ -100,24 +113,50 @@ describe("CompanyRepository", () => {
   });
 
   it("should rollback transaction and throw error if photos is null", async () => {
-    await expect(
-      CompanyRepository.create({ ...companyDataWithMinimumData, photos: [null] })
-    ).rejects.toThrow("aggregate error");
+    try {
+      await CompanyRepository.create({ ...companyDataWithMinimumData, photos: [null] });
+    } catch (aggregateError) {
+      expect(aggregateError).toBeInstanceOf(AggregateError);
```

Comment:
https://everyday.codes/javascript/create-custom-jest-matchers-to-test-like-a-pro/

y si hacemos algo asi?

```
expect(aggregateError).toBeAggregateErrorIncluding(
  ValidationError,
  "notNull Violation: CompanyPhoto.photo cannot be null"
)
```

Reply:
Me gusta! pero si prefieren meterlo en otro pr no me molesta tampoco